app/models/admin.rb

```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
I wanted to discuss this section related to Admin: we will create Admin credential on the Rails console, it seems to me proper to remove some abilities from here, for example:
- :registerable - we will not allow Admin to be registered on the web interface;
- :recoverable - the same;